sdg/components.py

```diff
@@ -22,22 +22,22 @@ def git_clone_op(
             # Increase logging verbosity
             set -x &&
```

Comment:
Optional: Can we not use `set -e` here rather tie everything together with `&&`?